### PR TITLE
Add Travis CI integration for Android

### PR DESCRIPTION
**Pull request by mikolak**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,7 +28,7 @@ _android_job_template: &android_job_template
       - $HOME/.gradle/wrapper/
   script:
     - cd android
-    - ./gradlew checkstyle assembleRelease test
+    - ./gradlew assembleRelease test
```
Comment (bartoszwilk) [inline .travis.yml]: What about just ./gradlew buildRelease? According to docs:
build: Assembles and tests this project.

File: .travis.yml [hunk 2]
```diff
@@ -23,6 +17,8 @@ _android_job_template: &android_job_template
   env:
     global:
       - GRADLE_OPTS="-Xms128m"
+      - ABI="default;armeabi-v7a"
+      - ADB_INSTALL_TIMEOUT=10 # minutes
```
Comment (bartoszwilk) [inline .travis.yml]: Don't we want do decrease this timeout threshold a bit?

Comment (mikolak) [reply .travis.yml]: This line can be completely removed, we're not launching an emulator - my bad.

File: .travis.yml (hunk 1 above)
Comment (mikolak) [reply .travis.yml]: Nice catch!

**Pull request closed without merging**